repos/fdbt-site/src/pages/products/exports.tsx

```diff
@@ -52,21 +53,25 @@ const getTag = (exportDetails: Export): ReactElement => {
     );
 };
 
-const Exports = ({ csrf, operatorHasProducts }: GlobalSettingsProps): ReactElement => {
+const Exports = ({ csrf, exportStarted, operatorHasProducts }: GlobalSettingsProps): ReactElement => {
     const [showExportPopup, setShowExportPopup] = useState(false);
     const [showFailedFilesPopup, setShowFailedFilesPopup] = useState(false);
     const [buttonClicked, setButtonClicked] = useState(false);
+    const [isExportStarted, setIsExportStarted] = useState(exportStarted);
 
     const { data } = useSWR('/api/getExportProgress', fetcher, { refreshInterval: 1500 });
 
+    setInterval(() => setIsExportStarted(false), 1000 * 1500);
```

Comment:
What is the intention of this line of code?

Reply:
to set the state of flag to false after 1500 secs